src/wb-git-status.sh

```diff
@@ -24,7 +24,7 @@ REVIEW_STATUS=""
 ISSUE_STATUS=""
 BUG_STATUS=""
 
-if [[ -z "$BRANCH" ]]; then
+if [[ -z $BRANCH ]]; then
```

Comment:
This is simple and we can just overlook, but try to avoid changes outside the scope of the issue

Reply:
Understood. That linting fix was applied in the BATS PR, and has been removed from this PR after rebasing.